rllib/tuned_examples/appo/multi-agent-cartpole-crashing-restart-env-appo.yaml

```diff
@@ -1,53 +0,0 @@
-multi-agent-cartpole-crashing-appo:
```

Comment:
We will move away from supporting restarting individual vector envs as we are moving to gym.vector.Env anyways. We could only support restarting individual envs if we remained in RLlib's VectorEnv API.